test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -19,6 +19,7 @@
                                                    paged-results*
                                                    deftestseq
                                                    parse-result]]
+            [puppetlabs.puppetdb.testutils.services :refer [create-db-map]]
```

Comment:
no, I had to move that function out of fixtures to where it is now. Previously it was getting sucked in by the :refer :all